lib/config-validator.ts

```diff
@@ -20,7 +20,8 @@ async function validate(
   desc: string,
   config: RenovateConfig,
   strict: boolean,
-  isPreset = false
+  isPreset = false,
+  isFileConfig = false
```

Comment:
```suggestion
  isGlobalConfig = false
```
more clear what that means. presets are also files configs.

Reply:
why resolved without a comment? 😕

Reply:
My bad...I think the change didn't get saved before staging and commiting